pkg/service/util.go

```diff
@@ -138,7 +138,8 @@ func HostsToSet(hosts []*models.Host) sets.Set[string] {
 	return hostSet
 }
 
-func GetPlacementGroupVMSet(placementGroup *models.VMPlacementGroup) sets.Set[string] {
+// GetVMsInPlacementGroup returns the virtual machine ID set in the placement group.
```

Comment:
// GetVMsInPlacementGroup returns a Set of IDs of the virtual machines in the placement group.